.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
       - run: npm install
-      - run: npm run test:unit
+      - run: npm run test
```

Comment:
If we are removing the react support you also have to adjust the pipelines for insider-release and for the release. Otherwise I expect those pipelines to fail...

Reply:
You are right. Fixed.